frontend/src/test/scala/bloop/bsp/BspClientTest.scala

```diff
@@ -70,7 +70,11 @@ object BspClientTest {
         case bsp.LogMessageParams(bsp.MessageType.Warning, _, _, msg) => logger.warn(msg)
         case bsp.LogMessageParams(bsp.MessageType.Error, _, _, msg) => logger.error(msg)
       }
-      .notification(endpoints.Build.publishDiagnostics) {
+
+    // Lsp4s fails if we try to repeat a handler for a given notification
```

Comment:
I was actually wondering if lsp4s should permit multiple notification handlers for the same endpoint, what do you think? The server would invoke all of the registered handlers in that case.

Reply:
> I was actually wondering if lsp4s should permit multiple notification handlers for the same endpoint, what do you think? The server would invoke all of the registered handlers in that case.
  
It would be a nice improvement in the library down the road :smile: